Pod/Classes/BONText.m

```diff
@@ -575,6 +575,15 @@ - (NSString *)debugString
     return [self debugStringIncludeImageAddresses:YES];
 }
 
+- (NSString *)debugStringLeftToRight
```

Comment:
#nth thinking this should be able to pass through image addresses, in case you need to use it for a unit test. Or would you always just call `-debugStringIncludeImageAddresses:NO` in that case? What do you think, @nevillco? Would the output of this method ever need to be deterministic?